performance/README.md

```diff
@@ -0,0 +1,114 @@
+# Howso Platform Performance Considerations
+
+## Overview
+
+This guide covers performance considerations for the Howso Platform and client installation. 
+
+## TLDR
+- Use a dedicated Kubernetes cluster for Howso Platform.
+- Increase the size of NATS, Redis, Postgres.
+- Use cluster autoscaling.
+- Run workloads from a large dedicated client machine.
+
+## Cluster Configuration
+
+### Dedicated Cluster
+
+In the Howso Platform, trainees (also known as workers) are created as new (stateful set) pods on demand.  
+- Individual trainees can be very large - it is not uncommon to dedicate tens of cpu resources and even more Gb of memory to a single trainee.
```

Comment:
"Use large nodes" seems like a concrete suggestion here; at least 16 cores and 64 GB RAM per node, for example.

Reply:
Fixed/added (but in the node pool section)